src/layouts/sidebar-sidecar/components/sidecar-scroll-container/sidecar-scroll-container.module.css

```diff
@@ -14,9 +14,7 @@
 	display: flex;
 	flex-direction: column;
 	isolation: isolate;
-
-	/* Note: max-sidecar-height is set in sidebar-sidecar-layout.module.css. */
-	max-height: var(--max-sidecar-height);
+	min-height: 0;
```

Comment:
`sidecar-scroll-container` is now within a flex-column layout, and the enclosing layout handles the max-height concern. We need `min-height: 0` here to ensure the `sidecar-scroll-container` starts at zero and grows to fill the flex column.